config/environments/production.rb

```diff
@@ -60,6 +60,7 @@
   # Precompile additional assets.
   # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
   config.assets.precompile += %w( mail.css colorbar.jpg logo.png intro_video.png )
+  config.assets.css_compressor = :sass
```

Comment:
I believe I was trying to ensure that the assets are referred to correctly if I embedded this in CSS as opposed to using any other asset path.  I can test to see if this is unnecessary.